app/serializers/api/v1/participant_serializer.rb

```diff
@@ -9,9 +9,18 @@ class ParticipantSerializer
       include JSONAPI::Serializer::Instrumentation
 
       class << self
+        def induction_record(profile, lead_provider)
+          # TODO: must be scopped to the lead provider
+          if profile.active?(lead_provider)
+            profile.current_induction_record
+          else
+            profile.induction_records.latest
```

Comment:
if there is a transfer and I am the old provider. The new provider makes some changes eg changes schedule which creates a new induction records. will the old provider not then see this new induction record even though it does not belong to them?